src/pages/Cafeteria/MobileCafeteriaPage/components/MobileMenuBlocks/index.tsx

```diff
@@ -0,0 +1,99 @@
+import useOnClickOutside from 'utils/hooks/useOnClickOutside';
+import { ReactComponent as CloseIcon } from 'assets/svg/close-icon.svg';
+import { Dining, MealType } from 'interfaces/Cafeteria';
+import useModalPortal from 'utils/hooks/useModalPortal';
+import { Portal } from 'components/common/Modal/PortalProvider';
+import { useEffect } from 'react';
+import useLogger from 'utils/hooks/useLogger';
+import { useDatePicker } from 'pages/Cafeteria/hooks/useDatePicker';
+import useDinings from 'pages/Cafeteria/hooks/useDinings';
+import MobileMealImage from 'pages/Cafeteria/MobileCafeteriaPage/components/MobileMealImage';
+import { MEAL_TYPE_MAP, PLACE_ORDER } from 'static/cafeteria';
+import styles from './MobileMenuBlocks.module.scss';
+
+interface Props {
+  mealType: MealType;
+}
+
+export default function MobileMenuBlocks({ mealType }: Props) {
+  const portalManager = useModalPortal();
+  const { target } = useOnClickOutside<HTMLImageElement>(portalManager.close);
+
+  const { currentDate } = useDatePicker();
+  const { dinings } = useDinings(currentDate);
+
+  const filteredDinings = dinings
+    .filter((meal) => meal.type === mealType)
+    .filter((meal) => !meal.menu.some((dish) => dish.name.includes('미운영')));
```

Comment:
두 번 filter를 사용하지 않고 `&&`으로 처리가능하려나요?
```suggestion
  const filteredDinings = dinings
    .filter((meal) => meal.type === mealType && !meal.menu.some((dish) => dish.name.includes('미운영')));
```

Reply:
네 가능할 것 같습니다! 제안해주신 방법이 원래 코드보다 더 덜 순회해서 조금 더 빠를 것 같아요!